devel/iniparser/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        ndevilla iniparser 4.1 v
+github.tarball_from archive
+revision            0
+
+categories          devel
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+
+license             MIT
+
+description         An ini file parser library for C
+
+long_description    ${description}
+
+patchfiles          Makefile.diff
+
+checksums           rmd160  06b85b087760fc40ab5a2568db663df25b13ccb6 \
+                    sha256  960daa800dd31d70ba1bacf3ea2d22e8ddfc2906534bf328319495966443f3ae \
+                    size    44732
+
+use_configure       no
```

Comment:
since you're not doing a `configure` stage you'll have to make sure yourself that you're [UsingTheRightCompiler](https://trac.macports.org/wiki/UsingTheRightCompiler) and that MacPorts' flags are being used. Currently that's not the case. In many case using the `makefile` PG can/will take care of most of this...

So please include that PG at the top of the Portfile:
``` 
PortGroup           github 1.0
```

that sets `use_configure no`, so you can remove this line here

Reply:
I've added the makefile PortGroup, thanks.

Reply:
... and removed the unnecessary `use_configure`.